streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -446,6 +446,12 @@ void print(final Serde<K> keySerde,
      * @param printed options for printing
      */
     void print(final Printed<K, V> printed);
+    
+    /**
+     * Merges the given streams into one larger stream. 
+     * @param a stream which is to be merged. However, there is no guarantee of order.
```

Comment:
`However...` should not be part of the parameter description. (If you use an IDE, it should show you the rendered JavaDoc if you hoover over the method. Otherwise, you can also build the JavaDocs via `./gradlew javadoc` and inspect in a browser.)

I would also rephrase a little bit. `There is no ordering guarantee between records from this {@code KStream} and records from the provided {@code KStream} in the merged stream. Relative order is preserved within each input stream though (ie, records within one input stream are processed in order).`

Reply:
Thanks, Matthias!